releasenotes/notes/deprecation-passmanager-0.25-95eb9b45b517370a.yaml

```
@@ -0,0 +1,4 @@
+---
+upgrade:
+  - |
+    Removed deprecated function :func:`~qiskit.transpiler.preset_passmanagers.common.get_vf2_limits` and its corresponding test.
```

```suggestion
    Removed deprecated function :func:`~qiskit.transpiler.preset_passmanagers.common.get_vf2_call_limit` and its corresponding test.
```

Typo in the function name, that's the replacement 😄.

---

We tend to also include one last reminder of what should be used instead for a removed deprecation in the upgrade notes. For example, adding something like "Instead, use :func:`~qiskit.transpiler.preset_passmanagers.common.get_vf2_limits` to the end of what you've already got.

---

Thanks for spotting these and for your suggestion! I updated the release note. :-)